x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -15,17 +16,22 @@ import * as euiVars from '@elastic/eui/dist/eui_theme_k6_light.json';
 
 import { AppFrontendLibs } from '../lib/lib';
 import { PageRouter } from '../routes';
+import { createStore } from '../store';
```

Comment:
Per our conversation from a few hours ago, this is the way that plugins operate where each one has a disjointed store. This is obviously not typically how you utilize redux store (single source of truth), but it does make sense here and with the current plugin model. Also, redux from our conversations will be more UI state and very little if any domain state (which will be Apollo)